p4-16/psa/PSA.mdk

```diff
@@ -1161,11 +1161,26 @@ corresponding table entries.
 [INCLUDE=psa.p4:ActionSelector_extern]
 ```
 
-## Packet Generation
+## Digest
+
+A PSA implementation provides a digest mechanism to send a separate
+data object to the control plane. The data object may contain a
+identifier to identify the receiver of the object. The data object can
+optionally attach metadata to provide more information about the
+source of the digest. The attached metadata can be one or more header
+fields from a packet or one or more metadata generated in the Ingress
+or Egress pipeline. The digest extern can only be invoked in the
+deparser.
+
+A common use case of the digest mechanism is MAC learning. In the MAC
+learning case, the receiver of the digest is the CPU port, represented
```

Comment:
why is it important here how is the CPU port represented? And actually `PortId_t` is not an enum.